metadata-ingestion/docs/sources/iceberg/iceberg.md

```diff
@@ -1,4 +1,175 @@
-### Concept Mapping
+## Setting up connection to an Iceberg catalog
+
+There are multiple servers compatible with the Iceberg Catalog specification. DataHub's `iceberg` connector uses `pyiceberg`
+library to extract metadata from them. The recipe for the source consists of 2 parts:
+
+1. `catalog` part which is passed as-is to the `pyiceberg` library and configures connection and its details (i.e. authentication).
+   The name of catalog specified in the recipe has no consequence, it is just a formal requirement from he library.
+   Only one catalog will be considered for the ingestion.
+2. Other part consists of parameters, such as `env` or `stateful_ingestion` which are standard DataHub's ingestor
```

Comment:
~~Other part~~ -> The remaining configuration consists...

"standard DataHub's **ingestion**"